grappa/reporters/subject.py

```diff
@@ -44,7 +43,7 @@ def run(self, error):
             # Render template
             text_message = self.render_tmpl(
                 self.indentify(text_message),
-                self.normalize(value)
+                value
```

Comment:
Instead of non-calling `normalize()` we should add the proper normalization logic for int/float types, don't you think?

Reply:
I removed it because it is already called in base reporter

```
def render_tmpl(self, tmpl, value):
        placeholders = {}

        if '{value}' in tmpl:
            placeholders['value'] = self.normalize(value)
```